src/main/java/org/traccar/WebDataHandler.java

```diff
@@ -54,15 +57,38 @@
     private final String header;
     private final boolean json;
 
+    private final Boolean retryEnabled;
+    private final Integer retryDelayMin;
+    private final Integer retryDelayMax;
+    private final Integer deliveryPendingLimit;
+
+    private AtomicInteger deliveryFaliuresInRow;
+    private AtomicInteger deliveryPendingCurrent;
+
     @Inject
     public WebDataHandler(
             Config config, IdentityManager identityManager, ObjectMapper objectMapper, Client client) {
+
         this.identityManager = identityManager;
         this.objectMapper = objectMapper;
         this.client = client;
         this.url = config.getString(Keys.FORWARD_URL);
         this.header = config.getString(Keys.FORWARD_HEADER);
         this.json = config.getBoolean(Keys.FORWARD_JSON);
+
+        this.retryEnabled = config.getBoolean(Keys.FORWARD_RETRY_ENABLE);
+
+        Integer retryDelayMin = config.getInteger(Keys.FORWARD_RETRY_DELAY_MIN, 1);
+        this.retryDelayMin = (retryDelayMin > 0 && retryDelayMin < 3600) ? retryDelayMin : 1;
```

Comment:
I would recommend to remove the validations for value ranges. Don't see any good reason to have additional code for it. Configuration is intended for admin users, which should know what they are doing. For other configuration we never do validation, so also for consistency we should remove.

Reply:
Good.

Reply:
Do you think we should not check even for negative values?

Reply:
I don't think we need to. Configuration is for admin users and it's usually done once. I don't see why someone would enter negative value, unless they want to try and break their own server.

Reply:
Jajaja... That's a good one. Ok.